src/zzz_od/gui/view/home/home_interface.py

```
@@ -268,31 +266,23 @@ def stop(self):
-    def _run_impl(self):
-        self.need_update.emit(self.ctx.model_config.using_old_model())
-
-class CheckBannerRunner(CheckRunnerBase):
     def _run_impl(self):
-        if self.ctx.signal.reload_banner:
-            self.need_update.emit(True)
+        try:
+            result = self.check_func(self.ctx)
+            if result is not None:
+                self.need_update.emit(result)
+        except Exception as e:
```

Ruff (0.14.5)

Do not catch blind exception: `Exception`

(BLE001)